src/utils/replaceWithExactPath.js

```diff
@@ -0,0 +1,18 @@
+const which = require('which');
+
+function replace(command) {
+    return command
+        .split(/&&|;|\n/)
```

Comment:
`|` or `||` operators can also as  command separators
Also any of these operators used inside quotes should be ignored
also new lines after `\`  operator should be ignored

Reply:
Please explore these further for any more such exceptions

Reply:
Fixed

Reply:
Hi, please review the changes I have made. I had no idea shell had so many separators haha! Thanks!